lib/ResponseDefinitions.php

```diff
@@ -93,21 +93,24 @@
  *     conversation?: string,
  * }
  *
- * @psalm-type TalkChatMessage = array{
+ * @psalm-type TalkChatBaseMessage = array{
```

Comment:
We're not using TS definitions for messages widely in frontend yet, so would be fine to change it now to preferable names, as soon as you'll come to agreement

Reply:
I'd somehow still like to have the ProxyMessage exposed as a type, so that if we have something in the future that is not in Base or Normal, we don't have to retroactively introduce a subclass.
But also not sure where the `Room` was coming from.

So if psalm allows this, we should add a:
```
@psalm-type TalkChatProxyMessage = TalkChatBaseMessage
```

And somehow indicate that `TalkChatBaseMessage` should not be "instanciated"?
If that is not possible, is basically means we remove the Base (and are back to adjusting the current name)

Reply:
@nickvergessen I pushed a version that should be what you envisioned.